### PR TITLE
Update comments in Focus Zones

### PR DESCRIPTION
**Pull request by sophieH29**
Update comments in FocusZone, FocusTrapZone, AutoFocusZone

### CODE REVIEW

File: packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.types.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ export interface AutoFocusZoneProps extends React.HTMLAttributes<HTMLDivElement>
 
   /**
    * Indicates the selector for first focusable item.
+   * @default the first tabbable element will get focus.
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.types.tsx]: It should `undefined`?

```diff
-   * Indicates the selector for first focusable item.
-   * @default the first tabbable element will get focus.
+  Indicates the selector for first focusable item, will use the first tabbable element will get focus.
```

File: packages/react/src/lib/accessibility/FocusZone/FocusZone.types.ts [hunk 2]
```diff
@@ -32,111 +32,137 @@ export interface IFocusZone {
...
-   * Function which uses root element to select and return the intial tabbable element.
+   * Function which uses root element as parameter to return the intial tabbable element.
+   * For example, when there is a chat with a bottom-up approach, it is expected that the last chat message is tabbable (active), not the first default one.
+   * @default undefined
    */
   defaultTabbableElement?: (root: HTMLElement) => HTMLElement
 
   /**
    * If a default tabbable element should be force focused on FocusZone mount.
+   * @default false
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/FocusZone/FocusZone.types.ts]: I suggest to patch our plugin instead, something like `if (type === 'boolean' && default === undefined) default = false`

File: packages/react/src/lib/accessibility/FocusZone/FocusTrapZone.types.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ export interface FocusTrapZoneProps extends React.HTMLAttributes<HTMLDivElement>
 
   /**
    * Sets the HTMLElement to focus on when exiting the FocusTrapZone.
-   * @default The element.target that triggered the FTZ.
+   * @default The target which triggered the FocusTrapZone.
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/FocusZone/FocusTrapZone.types.tsx]: The similar to https://github.com/stardust-ui/react/pull/1289/files#r280799691

File: packages/react/src/lib/accessibility/FocusZone/FocusZone.types.ts (hunk 2 above)
Comment (sophieH29) [reply packages/react/src/lib/accessibility/FocusZone/FocusZone.types.ts]: sorry, didn't get 🙃

**Review by layershifter — APPROVED** [review]

**Pull request merged**